operate/client/src/modules/queries/auditLog/useAuditLogs.ts

```diff
@@ -37,6 +46,16 @@ const useAuditLogs = <T = QueryAuditLogsResponseBody>(
     select: options?.select,
     staleTime: 5000,
     refetchInterval: 5000,
+    initialPageParam: 0,
```

Comment:
@danielkelemen one thing to keep in mind is that if the user scrolls all the way the poll interval will make a request for each loaded page in sequence

I guessing this part of the UI won't be used as often, so I think it won't be a problem now but we definitely need to keep an eye out for that